R/mock-session.R

```diff
@@ -85,7 +85,8 @@ makeNoop <- function(name, msg = paste0(name, " is a noop.")) {
   body(impl) <- rlang::expr({
     # Force arguments
     !!lapply(formalArgs(impl), rlang::sym)
-    private$noopWarn(!!name, !!msg)
+    # Evade "no visible binding" note for reference to `private`
+    get("private")$noopWarn(!!name, !!msg)
```

Comment:
Instead of putting a `get()` in each function, we can put the symbol in there directly with something like:

```R
private_sym <- as.symbol("private")
rlang::expr({ !!private_sym })
```

An alternate way of doing this is by calling `globalVariables()` (though we use it elsewhere, so you'd need to use `add=TRUE`).

Reply:
I thought about the globalVariables() way, but I thought it might be more confusing than helpful... since `private` ever isn't actually global